test/util/helper.go

```diff
@@ -1,5 +1,3 @@
-//go:build unit
```

Comment:
instead of getting rid of it because integration needs it, could we do the `//go:build unit || integration`?

Reply:
Didn't even know this is possible. Thanks